pkg/machines/machines.scss

```diff
@@ -4,7 +4,7 @@
 @import "../../node_modules/@patternfly/patternfly/components/Page/page.css";
 @import "../lib/ct-card.scss";
 
-#app .pf-c-card {
+.ct-vm-overview .pf-c-card {
```

Comment:
There are cards also in the storage pages and network pages with this PR, which need this rule.